lib/galaxy/jobs/runners/cli.py

```diff
@@ -177,6 +180,9 @@ def __get_job_states(self):
         for job_destination_id, v in job_destinations.items():
             job_destination = v['job_destination']
             job_ids = v['job_ids']
+            if not job_ids:
```

Comment:
Hmm, can't really reproduce it now and indeed, job_ids must always be a list. Worst case would be a list that is filled with `None`, but this wouldn't be caught by this check anyway. I'll revert the commit and if it happens again I can check what is going on exactly.